authn-oidc-files/policy.yml

```diff
@@ -14,6 +14,9 @@
   - !variable
     id: provider-uri
 
+  - !variable
```

Comment:
I believe this var is defined twice in the same policy

Reply:
Yes, we have 2 policies in this file - one for okta and one for keycloak